### DISCUSSION TITLE
How to use Pixi Viewport plugin with react-pixi-fiber?

### DISCUSSION THREAD

**Discussion by saricden** [michalochman/react-pixi-fiber#278] [Q&A] [answered]
Hi there,

I'm looking to use the `pixi-viewport` plugin in my app running `react-pixi-fiber`.
This is the plugin: https://github.com/davidfig/pixi-viewport

Any ideas where I would start for this?

Ideally I'd like to be able to use it something like this:
```js
<Viewport>
  <CustomSpriteComponent1 />
  <CustomSpriteComponent2 />
  <CustomSpriteComponent3 />
</Viewport>
```

**Comment by michalochman** [comment, marked as answer]
Hi @saricden, please try this comment https://github.com/michalochman/react-pixi-fiber/issues/42#issuecomment-460757076
